pom.xml

```diff
@@ -32,7 +31,7 @@
 	        <dependency>
 	            <artifactId>mule-api</artifactId>
 	            <groupId>org.mule.runtime</groupId>
-	            <version>1.1.1</version>
+	            <version>${mule.api.version}</version>
```

Comment:
This version is defined in the parent pom and should not be redefined here (specially since it is using the same value).
property value: [here](https://github.com/mulesoft/mule-extensions-parent/blob/f35316fa2e60668015bd3b6fb67c284f2f43f64c/pom.xml#L114C1-L115C1)
dependency declaration: [here](https://github.com/mulesoft/mule-extensions-parent/blob/f35316fa2e60668015bd3b6fb67c284f2f43f64c/pom.xml#L165C14-L165C14)